tests/ledger.py

```diff
@@ -109,24 +112,43 @@ def __init__(self, filename):
     def __del__(self):
         self._file.close()
 
+    def upack_flatbuffer(self, data):
```

Comment:
Should this be `unpack`?

Reply:
the name? not necessarily, it could be `strip`, or `remove`

Reply:
It's currently `upack`, with no n. Is this a typo for u**n**pack_flatbuffer?

Reply:
ah I didn't see that. yes it should be `unpack`